src/configurator.cpp

```diff
@@ -30,6 +31,16 @@ const char *kMyComputerNamespacePath =
     "Software\\Microsoft\\Windows\\CurrentVersion"
     "\\Explorer\\MyComputer\\Namespace";
 
+const char* const kPreconfigureDirectory = "PreconfigureDirectory";
+QString getPreconfigureDirectory()
+{
+    RegElement reg(HKEY_CURRENT_USER, "SOFTWARE\\Seafile", kPreconfigureDirectory, "");
```

Comment:
I see the key is "SOFTWARE\\Seafile", isn't windows registry key case sensentive? (I'm not sure)